### PR TITLE
Update README to fix bugs, add links, and update structure

### PR DESCRIPTION
**Pull request by pabloest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+Welcome! This repository is the standard library for open source [Semgrep](https://semgrep.dev/) rules.
 
-Welcome! This repository is the standard library for [Semgrep](https://semgrep.dev/) rules. There are many more rules available in the [Semgrep Registry](https://semgrep.dev/explore) written by [Semgrep, Inc.](https://semgrep.dev/) and other contributors. The [Semgrep Registry](https://semgrep.dev/explore) includes rules from this repository and additional rules that are accessible within [Semgrep Cloud Platform](https://semgrep.dev/pricing). If there is a specific rule you are looking for, see the [Semgrep registry search](https://semgrep.dev/r). To contribute, find details about contributing in the [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) documentation.
+In addition to rules in this repository, the [Semgrep Registry](https://semgrep.dev/explore) also includes proprietary rules that enable interfile and interprocedural analsis, called [Pro rules](https://semgrep.dev/products/semgrep-code/pro-rules).
```
Comment (khorne3) [inline README.md]:
```suggestion
In addition to the rules in this repository, the [Semgrep Registry](https://semgrep.dev/explore) offers proprietary [Pro rules](https://semgrep.dev/products/semgrep-code/pro-rules) that enable interfile and interprocedural analysis.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
 
-## Using Semgrep rules repository
+- Find rules: search for open source and Pro rules through the [Semgrep registry search](https://semgrep.dev/r).
+- Use rules: Scan your code with these rules through the [Semgrep AppSec Platform](https://semgrep.dev/login)
```
Comment (khorne3) [inline README.md]:
```suggestion
- Use rules: Scan your code with these rules through [Semgrep AppSec Platform](https://semgrep.dev/login)
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
+- Contribute to rules: see the [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) documentation.
```
Comment (khorne3) [inline README.md]:
```suggestion
- Contribute to rules: see [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) for more information.
```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
 
-Run existing and custom Semgrep rules locally with the Semgrep command line interface (Semgrep CLI) or continuously with Semgrep in CI while using Semgrep App. To start using Semgrep rules, see [Semgrep tutorial](https://semgrep.dev/learn), [Getting started with Semgrep CLI](https://semgrep.dev/docs/getting-started/), and [Getting started with Semgrep App](https://semgrep.dev/docs/semgrep-app/getting-started-with-semgrep-app/).
+## Using the Semgrep rules repository
+
+Run existing and custom Semgrep rules locally with the Semgrep command line interface (Semgrep CLI) or continuously with Semgrep in CI while using Semgrep AppSec Platform. To start using Semgrep rules, see [Semgrep tutorial](https://semgrep.dev/learn), [Getting started with Semgrep CLI](https://semgrep.dev/docs/getting-started/), and [Getting started with Semgrep App](https://semgrep.dev/docs/semgrep-app/getting-started-with-semgrep-app/). To write your own Semgrep rules, see the [Semgrep tutorial](https://semgrep.dev/learn) and [documentation on writing rules](https://semgrep.dev/docs/writing-rules/overview/).
```
Comment (khorne3) [inline README.md]:
```suggestion
To start writing and using Semgrep rules, see [Learn Semgrep syntax](https://semgrep.dev/learn) and [Writing rules](https://semgrep.dev/docs/writing-rules/overview/). Then, run existing and custom Semgrep rules locally with the [Semgrep command line interface (Semgrep CLI)](https://semgrep.dev/docs/getting-started/) or [continuously with Semgrep in CI while using Semgrep AppSec Platform](https://semgrep.dev/docs/semgrep-app/getting-started-with-semgrep-app/).
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,34 @@
...
+
+## Writing Semgrep rules
+
+See Semgrep documentation on [writing rules](https://semgrep.dev/docs/writing-rules/overview/), including:
```
Comment (khorne3) [inline README.md]:
```suggestion
See [Writing rules](https://semgrep.dev/docs/writing-rules/overview/) for information including:
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,34 @@
...
+
+- Pattern syntax, describing what Semgrep patterns can do in detail, and provides example use cases of the ellipsis operator, metavariables.
```
Comment (khorne3) [inline README.md]:
```suggestion
- Pattern syntax, describing what Semgrep patterns can do in detail, and example use cases of the ellipsis operator, metavariables.
```

**Review by khorne3 — APPROVED** [review]
Just some small things

**Comment by pabloest** [comment]
Thanks for the review, @khorne3. I accepted your suggestions. Can you please check it again?

**Review by khorne3 — COMMENTED** [review]
LGTM!

**Review by khorne3 — APPROVED** [review]

**Comment by pabloest** [comment]
@khorne3 I think you have to re-enter your review in GH so I just re-requested that.

**Review by p4p3r — APPROVED** [review]
Lgtm

**Review by khorne3 — APPROVED** [review]

**Pull request merged**